### ISSUE TITLE
False positive of font-family SCSS variables for value-keyword-case

### ISSUE THREAD

**Issue by igorhrcek** [oblakstudio/wpwebpack#3]
**Describe the bug**
There are false positives for `font-family` value of SCSS variable.

**To Reproduce**
1. Add the following code to any SCSS file:
```
$font-family-base: "Manrope", system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
```
2. Run `npm run watch`
3. Observe console error

**Expected behavior**
Font families should be ignored

**Additional context**
Update `.stylelintrc` with these possible solutions:
1. Ignore some font families:
```
{
  "rules": { "value-keyword-case": ["lower", { "ignoreKeywords": ["BlinkMacSystemFont"] }] }
}
```

2. Ignore all families (this example is for Bootstrap 5):
```
{
  "rules": { "value-keyword-case": ["lower", { "ignoreKeywords": [$font-family-base] }] }
}
```

3. Ignore all variables:
```
{
  "rules": { "value-keyword-case": ["lower", { "ignoreProperties": ["/^\\$/"] }] }
}
```

**Comment by seebeen** [comment]
Hello,

I've read the issues in stylelint repo (namely [#4622](https://github.com/stylelint/stylelint/issues/4622) and [#1101](https://github.com/stylelint/stylelint/issues/1101).

I've added the following options to the .stylelintrc file:

```json
{
  "value-keyword-case": [
    "lower",
    {
      "ignoreKeywords": [
        "BlinkMacSystemFont"
      ],
      "ignoreProperties": [
        "$main_font_family",
        "$font-family-base"
      ]
    }
  ]
}
```

Feel free to other fonts and variable names as needed.

Issue will be fixed in the next fix commit.

**Comment by seebeen** [comment]
Fixed in 2.0.1
